### DISCUSSION TITLE
How to add Email Address that can send or recieve mail through Postal

### DISCUSSION THREAD

**Discussion by praction1988** [postalserver/postal#2611] [Help with using Postal] [answered]
I have a common question, but I couldn't find information on how to configure an email address that can send emails through a postal SMTP server and receive emails on it. Can you please provide guidance on this?

**Comment by willpower232** [comment]
Postal does not manage inboxes, it mostly enables software to send messages.

If you wanted to have an inbox approach, you might be able to rig something up with the routes functionality but it wouldn't be ideal.

Are you trying to use something like Outlook or Thunderbird to send email or are you trying to make a website or printer send emails?

**Comment by praction1988** [comment]
Hey,

Thank you for your prompt response. I was wondering how that "Receiving e-mail by HTTP" message will operate if there is no inbox available with Postal.

Is there another procedure or DOC that will make this easier to understand?

**Reply by willpower232** [reply]
ah I see what you mean, unfortunately the routing feature appears to be missing from the documentation.

If you go into the Routing section you can add a HTTP endpoint which links to your application then you can add a route and you'll see the first field resembles an email address

![image](https://github.com/postalserver/postal/assets/1619102/048ce701-eaa0-461e-bf1f-f6ecf6679c0b)

**Reply by praction1988** [reply]
Thank you for your prompt response to and comprehensive instructions. I'll put it to the test and report back.

If I may, I have one more question. is there extensive information on how to enable STARTTLS for the postal SMTP server?

**Reply by willpower232** [reply, marked as answer]
I'm not sure if it is documented but you basically have to provide the certificate files, change a value to true, and then restart Postal

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/config/postal.defaults.yml#L69-L71

**Reply by praction1988** [reply]
Thanks @willpower232  appricated. It's work. we can close this query.
